From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Thu, 5 Nov 2015 20:46:08 +0100
Subject: [PATCH] {MAYBE UPS} (NEW in 4.4) drivers/staging/wilc1000: remove
 wilc_debug_func type

This type is not used anywhere and lacks a __printf attribute
---
 drivers/staging/wilc1000/wilc_wlan.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/wilc1000/wilc_wlan.h b/drivers/staging/wilc1000/wilc_wlan.h
index de6c4ddbf45a..66771853f2da 100644
--- a/drivers/staging/wilc1000/wilc_wlan.h
+++ b/drivers/staging/wilc1000/wilc_wlan.h
@@ -199,7 +199,7 @@
  *      Debug Type
  *
  ********************************************/
-typedef void (*wilc_debug_func)(u32, char *, ...);
+/* typedef __printf(2, 3) void (*wilc_debug_func)(u32, const char *, ...); UNUSED? */
 
 /********************************************
  *
-- 
